docker/multi-process/Dockerfile

```diff
@@ -12,29 +12,33 @@ ENV HOME=/app
 
 ARG UID=1001
 RUN useradd -u "$UID" -g 0 -d /app -s /sbin/nologin -c "default user" default
+RUN chown -R "$UID:0" /app
+USER $UID
 
 ENV LC_ALL=en_US.UTF-8
 ENV RAILS_ENV=production
 
-COPY ["Gemfile", "Gemfile.lock", "/app/"]
-COPY lib/gemfile_helper.rb /app/lib/
-COPY vendor/gems/ /app/vendor/gems/
+COPY --chown="$UID:0" ["Gemfile", "Gemfile.lock", "/app/"]
+COPY --chown="$UID:0" lib/gemfile_helper.rb /app/lib/
+RUN mkdir /app/vendor
+COPY --chown="$UID:0" vendor/gems/ /app/vendor/gems/
+
+ARG ADDITIONAL_GEMS=
+ENV ADDITIONAL_GEMS=$ADDITIONAL_GEMS
```

Comment:
Nice idea, this allows one to bake agent gems into their docker image to avoid the additional bundle install time at startup?

Reply:
Yes, actually I didn't notice that the startup process ran bundle install, and building my own Docker image has been how I build and use Huginn, and thought this should be everybody's option. 😊